src/app/pages/home/home.component.ts

```diff
@@ -9,42 +9,42 @@ import { Router } from '@angular/router';
 
 export class HomeComponent implements OnInit {
     posts: object;
-  
-    post:{
-        id: number,
-        title: string,
-        username:string,
-        description: string,
-        photo: string,
-        post_status_id: number,
-        post_priority_id: number,
-        city: string,
-        state: string,
-        zip_code: string,
-        email: string,
-        customer_id:number,
-        customerId:string,
-        can_bid: boolean,
-        budget: number,
-        category_id: number,
-      } = {
-        id:-1,
-        title:'',
-        description:'',
-        username:'',
-        photo:'',
-        post_status_id: null,
-        post_priority_id: null,
-        city:'',
-        state:'',
-        zip_code:'',
-        email:'',
-        customer_id:null,
-        customerId:null,
-        can_bid:false,
-        budget:null,
-        category_id: null,
-      }
+    post: any;
```

Comment:
I think you missed this one? Or are you letting Jenn take care of this one?

Reply:
Jenn will take care of these